server/timestamp/timestamp.go

```diff
@@ -130,13 +130,14 @@ func CreateTimestamp(gun string, prev *data.SignedTimestamp, snapshot []byte, st
 	if prev != nil {
 		ts.Signed.Version = prev.Signed.Version + 1
 	}
-	sgndTs, err := json.MarshalCanonical(ts.Signed)
+	var sgndTs json.RawMessage
+	sgndTs, err = json.MarshalCanonical(ts.Signed)
```

Comment:
If I don't do it this way, `sgndTs` has type `[]byte` and I have to do a really ugly type cast from `[]byte` to `*json.RawMessage` on the line that changed just below. This is only one line longer and at least in my opinion much cleaner.